examples/EnTT/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2015, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-EnTT)
+
+# DOCUMENTATION_START {
+# EnTT requires at minimum C++17
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
Should be set in toolchain:
* https://docs.hunter.sh/en/latest/overview/customization/toolchain-id.html#simple-toolchains

In Hunter CI testing some toolchains already have C++17 enabled. If lower is not supported then toolchain can be excluded.

Reply:
Done, pushing shortly